action/action.go

```diff
@@ -491,25 +491,41 @@ func (a *Action) WriteBack() error {
 
 // decodeAnyResourceFile takes a file path and decodes it into a slice of
```

Comment:
This is the same logic, just inside a for loop iterating over `matches` found by the globbing. If it is a single file without glob chars, it will just iterate once on that file.